### PR TITLE
docs(toh-6): copyedits

### PR DESCRIPTION
**Pull request by chalin**
- Remove statement that doesn't apply: "We recommend registering app-wide services in the root
  `AppModule` *providers*" - but that isn't being done.
- Adding parentheses after method names; a few were missed in the previous pass.
- `debounce()` --> `debounceTime()`
- Remove obsolete Dart prose.
- Fix heading indentation.
- Added missing closing `</span>`.
- Fix misnamed Jade variables.

cc @kapunahelewong

### CODE REVIEW

File: public/docs/ts/latest/tutorial/toh-pt6.jade [hunk 1]
```diff
@@ -34,13 +34,14 @@ block start-server-and-watch
...
-  npm start
+  code-example(language="sh" class="code-shell").
+    npm start
 
-:marked
-  This command runs the TypeScript compiler in "watch mode", recompiling automatically when the code changes.
-  The command simultaneously launches the app in a browser and refreshes the browser when the code changes.
+  :marked
+    This command runs the TypeScript compiler in "watch mode", recompiling automatically when the code changes.
+    The command simultaneously launches the app in a browser and refreshes the browser when the code changes.
```
Comment (chalin) [inline public/docs/ts/latest/tutorial/toh-pt6.jade]: Lines 37-42 belong to the Jade `block` above it. This readjusts the indentation.

**Comment by kapunahelewong** [comment]
Thank you @chalin!!

File: public/docs/ts/latest/tutorial/toh-pt6.jade [hunk 2]
```diff
@@ -388,8 +382,8 @@ block observables-section-intro
   But requests aren't always done only once.
   You may start one request,
   cancel it, and make a different request before the server has responded to the first request.
-  A *request-cancel-new-request* sequence is difficult to implement with *!{Promise}s*, but
-  easy with *!{Observable}s*.
+  A *request-cancel-new-request* sequence is difficult to implement with *!{_Promise}s*, but
+  easy with *!{_Observable}s*.
```
Comment (wardbell) [inline public/docs/ts/latest/tutorial/toh-pt6.jade]: Please remove `!{...}` variables

File: public/docs/ts/latest/tutorial/toh-pt6.jade [hunk 3]
```diff
@@ -407,9 +401,8 @@ block observables-section-intro
   Instead you return the *Observable* from the the `htttp.get()`,
   after chaining it to another RxJS operator, <code>map()</code>,
   to extract heroes from the response data.
-
   RxJS operator chaining makes response processing easy and readable.
-  See the [discussion below about operators](#rxjs-imports).
+  See the [discussion below about operators](#rxjs-imports).</span>
```
Comment (wardbell) [inline public/docs/ts/latest/tutorial/toh-pt6.jade]: This is a `</span>` end tag with out an opening `<span>`. Why did you add it?

File: public/docs/ts/latest/tutorial/toh-pt6.jade (hunk 1 above)
Comment (wardbell) [inline public/docs/ts/latest/tutorial/toh-pt6.jade]: But the block shouldn't be there at all! See summary.

File: public/docs/ts/latest/tutorial/toh-pt6.jade (hunk 3 above)
Comment (chalin) [inline public/docs/ts/latest/tutorial/toh-pt6.jade]: The span starts on line 406.

**Comment by chalin** [comment]
@wardbell - I agree with the goal of removing all blocks and variables, but this PR is only meant to "fix" a few issues. I'd rather the full cleanup be tackled in a follow-up PR.

File: public/docs/ts/latest/tutorial/toh-pt6.jade (hunk 3 above)
Comment (wardbell) [inline public/docs/ts/latest/tutorial/toh-pt6.jade]: You mean "starts on line 400". I see that now
